api/api/src/main/scala/uk/ac/wellcome/platform/api/rest/WorksController.scala

```diff
@@ -65,13 +67,17 @@ class WorksController(
           .findWorkById(id)(index)
           .flatMap {
             case Right(Some(work: IdentifiedWork)) =>
-              if (includes.collection) {
+              if (includes.anyRelation) {
+                retrieveRelatedWorks(index, work).map { relatedWorks =>
+                  workFound(work, relatedWorks, None, includes)
+                }
+              } else if (includes.collection) {
                 val expandedPaths = params._expandPaths.getOrElse(Nil)
-                retrieveTree(index, work, expandedPaths).map {
-                  workFound(work, _, includes)
+                retrieveTree(index, work, expandedPaths).map { tree =>
+                  workFound(work, None, tree, includes)
```

Comment:
with this number of arguments it might be worth naming the parameters as we pass them - otherwise we're descending into positional argument hell